packages/react/src/BubbleMenu.tsx

```diff
@@ -50,7 +50,7 @@ export const BubbleMenu = (props: BubbleMenuProps) => {
   }, [props.editor, currentEditor, element])
 
   return (
-    <div ref={setElement} className={props.className} style={{ visibility: 'hidden' }}>
+    <div ref={setElement} tabIndex={-1} className={props.className} style={{ visibility: 'hidden' }}>
```

Comment:
Did you test if a negative tab index still allows the client to focus elements inside the div via the keyboard?

Reply:
Yes, you can still focus on other elements inside

This is necessary so that `event.relatedTarget` is not `null` if an empty space was clicked inside the bubble-menu

Reply:
Yes, you can still focus on other elements inside using tab

This is necessary so that `event.relatedTarget` is not `null` if an empty space was clicked inside the bubble-menu